app/services/eligibility_service.rb

```diff
@@ -101,20 +110,18 @@ def presence_values
     presence
   end
 
-  def years_before(num_years)
-    (@day.to_date - num_years.year).strftime(date_format)
+  def years_before(day, num_years)
+    (day.to_date - num_years.year).strftime(EligibilityService.date_format)
   end
 
-  def date_format
+  def self.date_format
     '%Y-%m-%d'
   end
```

Comment:
Is this used anymore? We could probably use it on that duplicated line.

Reply:
Good catch. I've DRY-ed this.